drivers/adc/ad9083/ad9083_api/adi_ad9083_bf_dig_scan.h

```diff
@@ -0,0 +1,25 @@
+// SPDX-License-Identifier: GPL-2.0
+/*!
+ * @brief     SPI Register Definition Header File, automatically generated by
+ *            yoda2h v1.1.2 at 1/2/2020 9:13:37 AM.
+ *
+ * @copyright copyright(c) 2018 - Analog Devices Inc.All Rights Reserved.
+ *            This software is proprietary to Analog Devices, Inc. and its
+ *            licensor. By using this software you agree to the terms of the
+ *            associated analog devices software license agreement.
+ */
+
+/*!
+ * @addtogroup AD9083_BF
+ * @{
+ */
+#ifndef __ADI_AD9083_BF_DIG_SCAN_H__
+#define __ADI_AD9083_BF_DIG_SCAN_H__
+
+/*============= I N C L U D E S ============*/
+#include "adi_ad9083_config.h"
+
+/*============= D E F I N E S ==============*/
+
+#endif /* __ADI_AD9083_BF_DIG_SCAN_H__ */
+       /*! @} */
```

Comment:
nitpick: new line at the end of the file

Reply:
AD9083 API should not be changed